lib/logstash/inputs/http.rb

```diff
@@ -75,11 +83,20 @@ def register
   end # def register
 
   def run(queue)
```

Comment:
I think you can mount different rack handler with different endpoints.. I believe Rails support it that way.